chart/values.yaml

```diff
@@ -5,25 +5,14 @@ image:
 service:
   enabled: true                              # Creates a Kubernetes Service for the keptn-service-template-go
 
-distributor:
-  stageFilter: ""                            # Sets the stage this helm service belongs to
-  serviceFilter: ""                          # Sets the service this helm service belongs to
-  projectFilter: ""                          # Sets the project this helm service belongs to
+subscription:
   pubsubTopic: "sh.keptn.>"                  # Sets the events the service subscribes to
-  image:
-    repository: docker.io/keptn/distributor  # Container Image Name
-    pullPolicy: IfNotPresent                 # Kubernetes Image Pull Policy
-    tag: "0.17.0"                            # Container Tag
-  config:
-    queueGroup:
-      enabled: true                          # Enable connection via Nats queue group to support exactly-once message processing
 
 remoteControlPlane:
   enabled: false                             # Enables remote execution plane mode
   api:
     protocol: "http"                        # Used Protocol (http, https)
-    hostname: ""                            # Hostname of the control plane cluster (and Port)
-    apiValidateTls: true                     # Defines if the control plane certificate should be validated
```

Comment:
I'm still wondering if we need this "feature". Maybe @christian-kreuzberger-dtx does know a little bit more about that

Reply:
I removed `apiValidateTls` as it was not used anywhere and I could not find any reference in the `go-sdk`. The `hostname` on the other hand, is used to build the `KEPTN_API_ENDPOINT` in combination with the `protocol`
(e.g. `{{protocol}}://{{hostname}}/api`). 

Let's wait for @christian-kreuzberger-dtx  opinion here before merging.

Reply:
`apiValidateTls` was used to toggle SSL/TLS validation, e.g., in case of self-signed certificates.
Wondering if that feature is still available using go-sdk ...

Reply:
According to @warber this is still implement as part of the go-sdk, and you can just set
```
            - name: HTTP_SSL_VERIFY
              value: "{{ .Values.remoteControlPlane.api.apiValidateTls | default "true" }}"
```
(though not in the distributor, but in the primary container).

Reply:
I just checked and the functionality is still available. Therefore I added the Helm value to the values and deployment files.